controller/tap/server.go

```diff
@@ -105,8 +107,16 @@ func (s *server) TapByResource(req *public.TapByResourceRequest, stream pb.Tap_T
 	}
 
 	for _, pod := range pods {
+		// create the expected pod identity from the pod spec
+		name := fmt.Sprintf("%s.%s.serviceaccount.identity.%s.cluster.local", pod.Spec.ServiceAccountName, req.Target.Resource.Namespace, s.controllerNamespace)
```

Comment:
we're not consistent around this, but going forward recommend using getters when accessing protobuf structs, protects against nil pointers:
```go
req.GetTarget().GetResource().GetNamespace()
```